tests/examples/Fervo_Norbeck_Latimer_2024.txt

```diff
@@ -3,49 +3,70 @@
 ---   "Commercial-Scale Demonstration of a First-of-a-Kind Enhanced Geothermal System."
 ---   https://doi.org/10.31223/X52X0B
 
-Has Nonvertical Section, True
-Multilaterals Cased, True
-Well Geometry Configuration, 4,       ----L
-Number of Multilateral Sections, 2,  ---Two parallel horizontal sections
-Well Drilling Cost Correlation,3,				--- [-] Use built-in well drilling cost correlation #3 = vertical large diameter
-Horizontal Well Drilling Cost Correlation,4,				--- [-] Use built-in well drilling cost correlation #4 = deviated large diameter
-
+***Reservoir Parameters***
 Reservoir Model, 1
+Reservoir Volume Option,1,					---Should be 1 2 3 or 4. See manual for details.
+Reservoir Density, 2700
 Reservoir Depth, 2347 meter
-Number of Segments, 1
-Gradient 1, 68
-Number of Production Wells, 1
-Number of Injection Wells, 1
-Production Well Diameter, 7
-Injection Well Diameter, 7
-Ramey Production Wellbore Model, 1
-Injection Wellbore Temperature Gain, 0
-Production Flow Rate per Well, 63
-
+Reservoir Heat Capacity,1000,					---[J/kg/K]
+Reservoir Thermal Conductivity, 2.7
+Number of Fractures, 100
 Fracture Shape, 4
 Fracture Height, 100
 Fracture Width, 991 meter
```

Comment:
This 991 meter number comes from me stubbing in the distance between the 2 wells - I have no idea whether it could be representative. Did you get a chance to review this value?

Reply:
No, not yet, but that will happen